src/frontend/src/catalog/system_catalog/pg_catalog/pg_database.rs

```diff
@@ -68,6 +70,7 @@ pub fn new_pg_database_row(id: u32, name: &str) -> OwnedRow {
         Some(ScalarImpl::Bool(false)),
         Some(ScalarImpl::Bool(true)),
         Some(ScalarImpl::Int32(-1)),
+        Some(ScalarImpl::Int32(1663)),
```

Comment:
Based on https://www.postgresql.org/docs/current/catalog-pg-database.html, dattablespace refers to [pg_tablespace](https://www.postgresql.org/docs/current/catalog-pg-tablespace.html).oid. However, in our implemetation, pg_tablespace is an empty table. Will this be an issue? Can we set dattablespace to None instead?